types/swagger-schema-official/index.d.ts

```diff
@@ -182,40 +182,38 @@ export interface BasicAuthenticationSecurity extends BaseSecurity {
 export interface ApiKeySecurity extends BaseSecurity {
   type: 'apiKey';
   name: string;
-  in: string;
+  in: 'query' | 'header';
```

Comment:
do you have a link to specs that verifies this?

Reply:
Sure! https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#security-definitions-objec

---

![image](https://user-images.githubusercontent.com/1416224/58132108-0d755a80-7be6-11e9-8c19-bebeba1b990d.png)

Reply:
@mohsen1 Any feedback?